x-pack/heartbeat/monitors/browser/synthexec/synthexec.go

```diff
@@ -135,14 +130,22 @@ func runCmd(
 		cmd.Args = append(cmd.Args, "--match", filterJourneys.Match)
 	}
 
+	// Variant of the command with no params, which could contain sensitive stuff
+	loggableCmd := exec.Command(cmd.Path, cmd.Args...)
```

Comment:
can we move this inside if statement and do only if user has specified params. We could initialize it with loggableCmd as cmd till this point.

Reply:
We can't do that really since we use it further down outside the if.

Reply:
Think my comment wasn't clear, I meant to move the exec command creation only when required like this. 

```
	loggableCmd := cmd
	if len(params) > 0 {
		loggableCmd = exec.Command(cmd.Path, cmd.Args...)
		paramsBytes, _ := json.Marshal(params)
		cmd.Args = append(cmd.Args, "--params", string(paramsBytes))
		loggableCmd.Args = append(loggableCmd.Args, "--params", fmt.Sprintf("\"{%d hidden params}\"", len(params)))
	}
```